mmv1/third_party/terraform/services/bigquery/resource_bigquery_table_internal_test.go

```diff
@@ -576,16 +576,18 @@ func TestUnitBigQueryDataTable_schemaIsChangeable(t *testing.T) {
 func TestUnitBigQueryDataTable_schemaIsChangeableNested(t *testing.T) {
 	t.Parallel()
 	// Only top level column drops are changeable
-	skipNested := map[string]bool{"arraySizeDecreases": true, "typeModeReqToNullAndColumnDropped": true}
+	customNestedValues := map[string]bool{"arraySizeDecreases": false, "typeModeReqToNullAndColumnDropped": false}
 	for _, testcase := range testUnitBigQueryDataTableIsChangeableTestCases {
-		if !skipNested[testcase.name] {
-			testcaseNested := &testUnitBigQueryDataTableJSONChangeableTestCase{
-				testcase.name + "Nested",
-				fmt.Sprintf("[{\"name\": \"someValue\", \"type\" : \"INTEGER\", \"fields\" : %s }]", testcase.jsonOld),
-				fmt.Sprintf("[{\"name\": \"someValue\", \"type\" : \"INT64\", \"fields\" : %s }]", testcase.jsonNew),
-				testcase.changeable,
-			}
-			testcaseNested.check(t)
+		changeable, ok := customNestedValues[testcase.name]
```

Comment:
nit: This works but I think the reverse would be easier to understand - `changeable` is default to `testcase.changeable`, except when it's one of the two, then it's false.

Reply:
Done, changed it to a more readable form